### PR TITLE
feat(docs): add full examples for mongodb tasks

### PR DESCRIPTION
**Pull request by shrutimantri**

### CODE REVIEW

File: src/main/java/io/kestra/plugin/mongodb/Load.java [hunk 1]
```diff
@@ -33,13 +33,20 @@
...
+                namespace: company.team
+                
+                tasks:
+                  - id: load
+                    type: io.kestra.plugin.mongodb.Load
+                    connection:
+                      uri: "mongodb://root:example@localhost:27017/?authSource=admin"
+                    database: "my_database"
+                    collection: "my_collection"
+                    from: "{{ inputs.file }}"
```
Comment (wrussell1999) [inline src/main/java/io/kestra/plugin/mongodb/Load.java:48]: There is no input in this example

File: src/main/java/io/kestra/plugin/mongodb/InsertOne.java [hunk 2]
```diff
@@ -31,27 +31,41 @@
...
+                
+                tasks:
+                  - id: insertone
+                    type: io.kestra.plugin.mongodb.InsertOne
+                    connection:
+                      uri: "mongodb://root:example@localhost:27017/?authSource=admin"
+                    database: "my_database"
+                    collection: "my_collection"
+                    document:
+                      _id:
```
[the remark this answers is not in the document]
Comment (shrutimantri) [reply src/main/java/io/kestra/plugin/mongodb/InsertOne.java:47]: This is correct. This is something specific to mongodb.

**Review by wrussell1999 — CHANGES_REQUESTED** [review]
Add inputs to a few examples and it’s good to go

File: src/main/java/io/kestra/plugin/mongodb/Bulk.java [hunk 3]
```diff
@@ -44,13 +44,24 @@
...
-                "collection: \"my_collection\"",
-                "from: \"{{ inputs.file }}\"",
-            }
+            full = true,
+            code = """
+                id: mongodb_bulk
+                namespace: company.team
+                
+                inputs:
+                  - id: file
```
Comment (wrussell1999) [inline src/main/java/io/kestra/plugin/mongodb/Bulk.java:53]:
```suggestion
                  - id: myfile
```

File: src/main/java/io/kestra/plugin/mongodb/Bulk.java [hunk 4]
```diff
@@ -44,13 +44,24 @@
...
+                    type: FILE
+                
+                tasks:
+                  - id: bulk
+                    type: io.kestra.plugin.mongodb.Bulk
+                    connection:
+                      uri: "mongodb://root:example@localhost:27017/?authSource=admin"
+                    database: "my_database"
+                    collection: "my_collection"
+                    from: "{{ inputs.file }}"
```
Comment (wrussell1999) [inline src/main/java/io/kestra/plugin/mongodb/Bulk.java:63]:
```suggestion
                    from: "{{ inputs.myfile }}"
```

**Review by wrussell1999 — APPROVED** [review]

**Pull request merged**